content/songs.json

```diff
@@ -10,72 +10,629 @@
     "languages": [
         {
             "code": "en",
-            "label": "EN",
+            "label": "English",
             "default": true
-        },
-        {
-            "code": "jp",
-            "label": "日本語"
         }
     ],
     "hints": [
         {
-            "begin": 0.0,
-            "end": 2.0
+            "begin": 10.0,
+            "end": 12.0
         },
         {
-            "begin": 0.0,
-            "end": 5.0
+            "begin": 10.0,
+            "end": 15.0
         },
         {
-            "begin": 0.0,
-            "end": 8.0
+            "begin": 10.0,
+            "end": 18.0
         },
         {
-            "begin": 0.0,
-            "end": 12.0
+            "begin": 10.0,
+            "end": 22.0
         },
         {
-            "begin": 0.0,
-            "end": 20.0
+            "begin": 10.0,
+            "end": 30.0
         }
     ],
-    "autocomplete": [
-       "Plastic Love",
-       "Windy Summer",
-       "Bring Me To The Dancenight",
-       "Gone with the Sadness",
-       "Midnight Down Town",
-       "Overdose",
-       "Tuxedo Connection",
-       "Tobashite Taxi Man",
-       "Spice Road",
-       "Say Goodbye",
-       "I'm in Love",
-       "Older Girl",
-       "Sweet Love",
-       "Remember Summer Days",
-       "Driving My Love",
-       "Telephone Number"
-    ],
+    "autocomplete": [],
     "songs": [
         {
-            "src": "VEe_yIbW64w",
             "type": "youtube",
-            "name": "Stay With Me"
+            "src": "8UVNT4wvIGY",
+            "artist": "Gotye",
```

Comment:
this song list is great but will these all be loaded into the CMS when creating the game?

might be worth adding them to https://github.com/play-co/musicguesser
I had arman create that as a quick win/example for sharing as the "template" with a decent sized song list